app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,78 @@
+.baner {
+  color: white;
+  width: 100%;
+  height: 600px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-image: image_url("if.jpg");
+}
+
+.light {
+  color: white;
+}
+
+.center{
+  position: relative;
+  left: 50%;
+  margin-top: 150px;
+  transform: translateX(-50%);
+  a {
+    text-decoration: none;
+    color: white;
+  }
+  a:hover {
+    color: whitesmoke;
+  }
+}
+
+.center:hover {
+  background-color: dodgerblue;
+  border-color: dodgerblue;
+}
+
+.nav-pills, .feed {

```

Comment:
too global change. you are overriding all nav-pills in whole app
add ID or class to your element and specify CSS classes exactly to it